api/src/api/externalEndpointRouter.test.ts

```diff
@@ -56,7 +57,7 @@ describe("externalEndpointRouter", () => {
 
   afterAll(async () => {
     await new Promise((resolve) => {
-      return setTimeout(resolve, 500);
+      return setTimeout(resolve, StatusCodes.INTERNAL_SERVER_ERROR);
```

Comment:
[boulder] This is a little confusing because we're talking about promises, and the number 500, but the `setTimeout` function isn't looking at an API response. It's waiting for 500 milliseconds before proceeding. In this case, yes, we're functionally achieving the same thing, but it's not an internal server error.

Reply:
I see multiple instances of this change. I'm not going to comment on all of them because that feels excessive, but they should all be changed.